README.md

```diff
@@ -1,4 +1,4 @@
-<p align="center"><img alt="Differencify" src="http://i.imgur.com/D0Eapjx.png" width="150">
+<p align="center"><img alt="Differencify" src="http://i.imgur.com/0zwMggK.png" width="150">
```

Comment:
This is super inconsequential, so if you feel they're better off on an external host, it's fine.